src/v2/controllers/FormController.js

```diff
@@ -156,7 +161,22 @@ export default Controller.extend({
         .then(this.handleIdxSuccess.bind(this))
         .catch(error => {
           this.showFormErrors(this.formView.model, error);
+        })
+        .then(() => {
+          if (actionPath === 'cancel') {
+            // TODO: remove this line after OKTA-405474
+            if (useInteractionCodeFlow) {
+              this.options.settings.set('useInteractionCodeFlow', true);
+            }
+            this.options.appState.trigger('interactionCanceled', this.constructor);
+          }
         });
+    } else if (actionPath === 'cancel') {
```

Comment:
Is there real use case for this code branch? 
IIRC, There is no way that widget triggers `cancel` action but `actions[cancel]` is not function.

Reply:
Reverted these changes. Will move fix for expired session to another PR